src/QA/QA.Web/ViewModels/WithdrawVacancy/FindVacancyEditModel.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Esfa.Recruit.Qa.Web.ViewModels.WithdrawVacancy
+{
+    public class FindVacancyEditModel
+    {
+        public string VacancyReference { get; set; }
```

Comment:
long maybe?

Reply:
This is a freetext field so the user could type any garbage.